miio/integrations/roidmi/vacuum/roidmivacuum_miot.py

```diff
@@ -558,7 +558,6 @@ def status(self) -> RoidmiVacuumStatus:
         return RoidmiVacuumStatus(
             {
                 prop["did"]: prop["value"] if prop["code"] == 0 else None
```

Comment:
```suggestion
                prop["did"]: prop["value"] if prop["code"] == 0 else None
                # max_properties limmit to 10 to avoid "Checksum error" messages from the device.
```